src/components/navigation/AdminNavMenu.jsx

```diff
@@ -17,7 +17,8 @@ const { Sider } = Layout;
 const AdminNavMenu = () => {
   const [collapsed, setCollapsed] = useState(false);
   const [menuTitle, setMenuTitle] = useState('Abound Food Care');
-
+  const { pathname } = useLocation();
+  console.log({ pathname });
```

Comment:
We might need this later for debugging purposes, but go ahead and add a todo comment above this line to remove it later before we ship it to AFC